subprojects/gradle/test-project/src/main/kotlin/com/avito/test/gradle/module/AndroidAppModule.kt

```diff
@@ -0,0 +1,139 @@
+package com.avito.test.gradle.module
+
+import com.avito.test.gradle.buildToolsVersion
+import com.avito.test.gradle.dir
+import com.avito.test.gradle.file
+import com.avito.test.gradle.files.InstrumentationTest
+import com.avito.test.gradle.files.androidManifest
+import com.avito.test.gradle.files.build_gradle
+import com.avito.test.gradle.kotlinClass
+import com.avito.test.gradle.kotlinVersion
+import com.avito.test.gradle.module
+import com.avito.test.gradle.sdkVersion
+import java.io.File
+
+class AndroidAppModule(
+    override val name: String,
+    override val packageName: String = "com.$name",
+    override val plugins: List<String> = emptyList(),
+    override val buildGradleExtra: String = "",
+    override val modules: List<Module> = emptyList(),
+    override val enableKotlinAndroidPlugin: Boolean = true,
```

Comment:
i don't like this flags, it has same purpose as `plugins: List<String>`, maybe we could make this API more clean: separate plugins block; but not in this PR for sure